cmd/eudico/template.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
let's call this file `genesis.json` maybe?

Reply:
Fixed